src/app/pages/articles/articles-list/article-pre-view/article-pre-view.component.scss

```diff
@@ -0,0 +1,57 @@
+@import "src/styles/variables";
+@import "src/styles/maps";
+@import "src/styles/mixins";
+
+.articlePreView {
+	width: 250px;
+	height: 350px;
+	background-color: $white;
+	padding: 20px;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	transition: 0.2s ease-in-out;
+	@include border-radius(medium);
+
+	&__title-image {
+		width: 250px;
+		@include border-radius(medium);
+	}
+
+	&__title {
+		@include headings(h2)
+	}
+
+	&:hover {
+		box-shadow: 0px 10px 15px 0px rgba(0, 0, 0, 0.2);
```

Comment:
If you want to reuse the box-shadow you could also move it into a map.
This time you probably don't need a separate mixin and function to get it.